src/views/About.vue

```diff
@@ -40,7 +37,7 @@
         >
           <v-flex>
             <amb-list-item
-              id="version-info"
+              id="version-element"
               :title="version"
               icon-name="alpha-v-circle-outline"
               subtitle="Release Version"
```

Comment:
@vickywane let's update the value from "Release Version" to "UI App Version" so it's clear what the version number relates to.